src/tests/nativeaot/SmokeTests/HardwareIntrinsics/X64Avx512.csproj

```diff
@@ -25,6 +25,17 @@ $(CLRTestBashPreCommands)
         fi
     fi
 ]]></CLRTestBashPreCommands>
+
+    <CLRTestBatchPreCommands><![CDATA[
+$(CLRTestBatchPreCommands)
+    powershell -Command "(Add-Type -MemberDefinition '[DllImport(\"kernel32.dll\")] public static extern int IsProcessorFeaturePresent(int ProcessorFeature);' -Name W -PassThru)::IsProcessorFeaturePresent(41)"
+    IF %ERRORLEVEL% EQU 0 (
+      echo No support for AVX512, test not applicable.
+      exit /B 0
+    )
+]]></CLRTestBatchPreCommands>
```

Comment:
Can you remind me what the problem here is?

All of these tests have `if (Avx512F.IsSupported) { Test } else { ValidateThrowsPNSE(); }`, by adding this we no longer have anything that validates these APIs throw as expected when unsupported. So it was intentional we left at least one platform capable of running those tests, regardless of what the hardware supported.

I would expect that we have some way to skip this filter so that at the very least outerloop jobs can continue testing this scenario.

Reply:
The test is compiler with `--instruction-set:avx512f,avx512f_vl,avx512bw,avx512bw_vl,avx512cd,avx512cd_vl,avx512dq,avx512dq_vl`. We assume RyuJIT is allowed to use these anywhere. So if the cpu doesn't support these, we fail runtime initialization. This test doesn't test dynamic light up, it tests IsSupported is constant true

Reply:
Ah, I see. I misread which test this was covering.